harness/determined/common/experimental/experiment.py

```diff
@@ -498,9 +498,11 @@ def list_checkpoints(
         if not sort_by:
             sort_by = checkpoint.CheckpointSortBy.SEARCHER_METRIC
 
-        def get_with_offset(offset: int) -> bindings.v1GetExperimentCheckpointsResponse:
+        def get_with_offset(
```

Comment:
Could you please explain to me why this change to `get_with_offset` was necessary?